### ISSUE TITLE
False positive: Missing reference: Base

### ISSUE THREAD

**Issue by jakobnissen** [julia-vscode/StaticLint.jl#368]
To reproduce this:
* Julia version 1.9.1 or 1.9.0 (it probably also works with earlier versions but I did not test)
* Latest version of Julia for VSCode (as of 2023-06-08)
* Create a new package `Foo` with the following content in `Foo/src/Foo.jl`:
```julia
module Foo

baremodule Flags
using Base: @enum
@enum Flag flag
end

end
```
* Add `StructTypes` v 1.10.0 to the Project.toml. The error does not occur if StructTypes is not in the Project.

The following lint is displayed (see attached file) 
![image](https://github.com/julia-vscode/StaticLint.jl/assets/23193950/ff7ca385-7dfe-415c-a1b6-392a88715777)

However, it works fine, as `Base` is always importable in baremodules:
```julia
julia> using Foo

julia> Foo.Flags.flag
flag::Flag = 0
```

**Comment by pfitzseb** [comment]
This is fixed now.
